config/samples/flinkoperator_v1beta1_flinkjobcluster.yaml

```diff
@@ -25,18 +25,19 @@ spec:
       ui: 8081
     resources:
       limits:
-        memory: "1024Mi"
-        cpu: "200m"
+        memory: "2048Mi"
+        cpu: "500m"
   taskManager:
     replicas: 2
     resources:
       limits:
-        memory: "1024Mi"
-        cpu: "200m"
+        memory: "2048Mi"
+        cpu: "500m"
   job:
     jarFile: ./examples/streaming/WordCount.jar
     className: org.apache.flink.streaming.examples.wordcount.WordCount
     args: ["--input", "./README.txt"]
     parallelism: 2
   flinkProperties:
     taskmanager.numberOfTaskSlots: "1"
+    taskmanager.memory.flink.size: "1024m"
```

Comment:
setting this is not required; for Flink >= v1.10 users should set `taskmanager.memory.process.size` but this setting is automatically derived based on `MemoryProcessRatio` which defaults to 90

Reply:
I test with Flink `1.14` and add this prop fix the issue.
`taskmanager.memory.process.size` not works

Reply:
Yeah, removed these props.
And I have verified that increase the resources make all thinks works~